web/src/main/webResources/WEB-INF/data/config/index/records.json

```diff
@@ -1220,6 +1246,14 @@
         "type": "text",
         "analyzer": "${es.index.analyzer.default}"
       },
+      "anyfre": {
```

Comment:
Maybe we could have a field with a similar structure as other multilingual fields for `any`
```json
"any": {
  "default": "",
  "langfre": "",
  "langger": ""
}
```
but not sure we can do `copy_to: "any.langfre"` after. Maybe something to test ? https://www.elastic.co/guide/en/elasticsearch/reference/current/copy-to.html does not provide example for this case.